container/container_windows.go

```diff
@@ -8,6 +8,11 @@ import (
 	"path/filepath"
 
 	containertypes "github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/pkg/system"
+)
+
+const (
+	containerInternalConfigsDirPath = `C:\ProgramData\Docker\internal\configs`
```

Comment:
In the current Windows container implementation, the container system drive is always C, and we cannot easily read the container environment from the host.